OMEdit/OMEditGUI/Animation/AbstractAnimationWindow.h

```diff
@@ -40,13 +40,30 @@
 #include <QLineEdit>
 #include <QComboBox>
 #include <QTimer>
+#include <QDoubleSpinBox>
+#include <QLabel>
 
 #include "FMUSettingsDialog.h"
 
 class VisualizerAbstract;
 class ViewerWidget;
 class Label;
 
+class QDoubleSpinBoxIndexed : public QDoubleSpinBox
```

Comment:
Maybe not a good idea to use Q initial for a custom class. Gives a feeling that it is a Qt builtin class.